base/applications/notepad/lang/es-ES.rc

```diff
@@ -8,11 +8,11 @@ BEGIN
     "^C", CMD_COPY
     "^B", CMD_SEARCH
     "^T", CMD_GOTO
-    "R", CMD_REPLACE, VIRTKEY, CONTROL
+    "^R", CMD_REPLACE, VIRTKEY, CONTROL
     "^N", CMD_NEW
     "^A", CMD_OPEN
     "^P", CMD_PRINT
-    "^S", CMD_SAVE
+    "^G", CMD_SAVE
```

Comment:
This seems unnatural (especially when I see that on spanish windows they are using Ctrl+Shift+S to save-as...)

Reply:
I know that it could sound "unnatural", but it's the way the text processors did in Spanish. I attach different Screenshots that probe it (in Notepad, Paint and LibreOffice Writer). 
![photo5789589132048183978](https://user-images.githubusercontent.com/27921286/78549564-de1e0b00-7802-11ea-94a6-2d3b9f157e21.jpg)

![Captura de pantalla de 2020-04-06 12-23-46](https://user-images.githubusercontent.com/27921286/78549470-bcbd1f00-7802-11ea-9f98-3b24edda9825.png)